src/modules/billing-plans/billing-plan.controller.ts

```diff
@@ -10,6 +11,7 @@ export class BillingPlanController {
   constructor(private readonly billingPlanService: BillingPlanService) {}
 
   @Post('/')
+  @UseGuards(SuperAdminGuard)
```

Comment:
You need to include ApiBearerAuth decorator as well

Reply:
done